docs/tools/todo.md

```
@@ -30,7 +31,7 @@ The params object **is** a single op, the discriminator and its fields live at t
 
 | Field | Type | Required | Description |
 | --- | --- | --- | --- |
-| `op` | `"init" | "start" | "done" | "rm" | "drop" | "append" | "view"` | Yes | Operation discriminator. |
+| `op` | `"init" | "start" | "done" | "rm" | "drop" | "append" | "view" | "pending"` | Yes | Operation discriminator. |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 4; Actual: 11; Too many cells, extra data will be missing

(MD056, table-column-count)